quickbooks_desktop_endpoint.rb

```diff
@@ -169,13 +169,13 @@ class QuickbooksDesktopEndpoint < EndpointBase::Sinatra::Base
           puts name
           puts collection.values.first.inspect
 
-          record = collection.values.first
+          records = collection.values.first
 
 
-          puts({connection_id: @config['connection_id'], flow: @config['flow'], record: record.inspect})
-          record = allow_only_whitelisted_fields(record.first.with_indifferent_access)
+          puts({connection_id: @config['connection_id'], flow: @config['flow'], records: records.inspect})
+          records = records.map{|record| allow_only_whitelisted_fields(record.with_indifferent_access) }
```

Comment:
@bcgianni could this have been the weird "array" thing with whitelisting? Since this looks like it's an array of records rather than 1 record?